docker/jupyter/custom/JupyterLockingExtension.js

```diff
@@ -0,0 +1,325 @@
+const dialog = require('base/js/dialog')
+const utils = require("base/js/utils")
+
+define(() => {
+    var modalOpen = false
+    var meta = {}
+
+    const syncIssueButtons = {
+        'Make a Copy': {
+            click: () => saveAs()
+        },
+        'Reload the workspace version and discard your changes': {
+            click: () => updateLocalCopyWithRemote()
+        }
+    }
+
+    const lockIssueButtons = {
+        'Run in Playground Mode': {
+            click: () => openPlaygroundMode(),
+            'class': 'btn-primary'
```

Comment:
Why does this need a class but not the others?

Reply:
Still pending discussion with Adrian on the styling of buttons outside of the lock modal. Didn't want to add styling when its possible for the UX to change. One of the other buttons may receive this class, but im not sure which one will be considered the 'primary'

Reply:
ok